xml/System.ServiceModel.Channels/ServiceChannelProxy.xml

```diff
@@ -1,1536 +1,1540 @@
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Boolean</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets a value indicating whether a call was done to a user interface to obtain credential information.</summary>
+        <value>
+          <see langword="true" /> if the <see cref="M:System.ServiceModel.IClientChannel.DisplayInitializationUI" /> method was called (or the <see cref="M:System.ServiceModel.IClientChannel.BeginDisplayInitializationUI(System.AsyncCallback,System.Object)" /> and <see cref="M:System.ServiceModel.IClientChannel.EndDisplayInitializationUI(System.IAsyncResult)" /> methods; otherwise, <see langword="false" />.</value>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Channels.ServiceChannelProxy&gt; instance is cast to an &lt;xref:System.ServiceModel.IClientChannel&gt; interface.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.IClientChannel.DisplayInitializationUI">
+      <MemberSignature Language="C#" Value="void IClientChannel.DisplayInitializationUI ();" />
+      <MemberSignature Language="ILAsm" Value=".method hidebysig newslot virtual instance void System.ServiceModel.IClientChannel.DisplayInitializationUI() cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.ServiceModel.Channels.ServiceChannelProxy.System#ServiceModel#IClientChannel#DisplayInitializationUI" />
+      <MemberSignature Language="VB.NET" Value="Sub DisplayInitializationUI () Implements IClientChannel.DisplayInitializationUI" />
+      <MemberSignature Language="F#" Value="abstract member System.ServiceModel.IClientChannel.DisplayInitializationUI : unit -&gt; unit&#xA;override this.System.ServiceModel.IClientChannel.DisplayInitializationUI : unit -&gt; unit" Usage="serviceChannelProxy.System.ServiceModel.IClientChannel.DisplayInitializationUI " />
+      <MemberSignature Language="C++ CLI" Value=" virtual void System.ServiceModel.IClientChannel.DisplayInitializationUI() = System::ServiceModel::IClientChannel::DisplayInitializationUI;" />
+      <MemberType>Method</MemberType>
+      <Implements>
+        <InterfaceMember>M:System.ServiceModel.IClientChannel.DisplayInitializationUI</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters />
+      <Docs>
+        <summary>A call to a user interface to obtain credential information.</summary>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Channels.ServiceChannelProxy&gt; instance is cast to an &lt;xref:System.ServiceModel.IClientChannel&gt; interface.</remarks>

```

Comment:
```suggestion
        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Channels.ServiceChannelProxy"/> instance is cast to an <see cref="T:System.ServiceModel.IClientChannel"/> interface.</remarks>

```